CIP-0011/README.md

```diff
@@ -69,6 +60,33 @@ reward address (with `network_id=1`)
 stake1uy8ykk8dzmeqxm05znz65nhr80m0k3gxnjvdngf8azh6sjc6hyh36
 ```
 
+## Rationale: how does this CIP achieve its goals?
+
+### Meaning of *account*
+
+The term "account" is unfortunately an overloaded term so we clarify all its uses here:
+
+#### 1) "Account" as a BIP44 derivation level
+
+BIP44 uses the term "account" as one derivation level to mean the following
+
+> This level splits the key space into independent user identities, so the wallet never mixes the coins across different accounts.
+To differentiate this from other usage, we sometimes refer to it as an `account'` (the bip32 notation) or a BIP44 Account.
+
+#### 2) "Account" as a transaction model
+
+Blockchains like Ethereum does not use the UTXO model and instead uses the [*Account model*](https://github.com/ethereum/wiki/wiki/Design-Rationale#accounts-and-not-utxos) for transactions.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [x] All notable wallet providers follow this method of key derivation.
+
+### Implementation Plan
+
+- [x] This method of key derivation has been agreed as canonical and made available to all wallet providers beginning with the Shelley ledger era.
```

Comment:
```suggestion
- [x] This method of key derivation has been agreed as canonical and has been included in [CIP-1852 | HD (Hierarchy for Deterministic) Wallets for Cardano](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/README.md).

- [x] This method of key derivation has been supported by all wallet and tool providers beginning with the Shelley ledger era.
```

I like included the add to CIP-1852 as it is an important step IMO

Reply:
yes, good addition & correction; have committed... but had to put inter-CIP link into some canonical form to make sure it works on the Dev Portal which I think will not parse the absolute links into links also on the Dev Portal.  Otherwise your changes are the same. 🙏